### DISCUSSION TITLE
will there ever be 3d tweens like adobe animate?

### DISCUSSION THREAD

**Discussion by BerzCookie** [keiwando/toonsquid#603] [Q&A]
because when i try to rotate things in a "3d" way it turns out sketchy and weird.

**Comment by Lioncub901** [comment]
My guess is that there has to be 3d support before we start thinking about 3d tweens. I think it might be done in the future. I don’t know.

**Comment by kavlor** [comment]
A simple Z axis (movement and rotation)might be helpful for making things move in and out of the foreground .  It would also be good for getting correct perspective of things like walls of buildings. Some very old animation apps had this Z axis so you could render rotating text to give a nice intro. There was even a vector animation app called Aegis animator that allowed pushing points into the z axis, this was before even Flash existed.

This was done in Aegis and if you watch you see certain parts are using a Z axis.
https://youtu.be/7bQJu97qxLw?t=74
